### ISSUE TITLE
MM for Gleev Weekly Calls 

### ISSUE THREAD

**Issue by dmtrjsg** [Joystream/gleev#29]
This is a meta issue to encapsulate the MM from the weekly Gleev calls

**Comment by dmtrjsg** [comment]
# MM 26.02

### YPP
- Would be nice to have report shared on Monday morning so we have freshest
- Report should be hosted on Notion
- Bedeho needs to be tagged on EOD stats in cases when Tier is  > Gold channels
- Internally some misalignment on what represents diff tiers

### JSG
- Nara QA is ongoing
- Feed of top 50 fixed videos is released
- Working on removing shorts

### Uploads
- Colossus bug is being investigated by BWG
- Fixed bug in Atlas which would chose diff storage provider on upload did not worn in full
Issue reported: https://discord.com/channels/811216481340751934/1210329640036597770/1210694112257376317 
Responsible @kdembler and Yassir

**Recommendation**
- Not much progress in the light of recent events. Likely to be done by Recombee going fwd. 

**Homepage feed**

Status
- Still ongoing issue
- Next Step will be to fix 50 videos shortlisted by Richard to the top of the page 
- After: review the weights of top videos and calibrate with the weights of english speaking channels

Action plan for feed

1. Update Orion with the video IDs that Richard shortlisted @Igrex
2. Deploy dev instance of Gleev with top 50 videos  @radek
3. Review if better or not @Bedeho 
4. If better deploy to production @Radek
5. To that version add a special URL for /feed-by-channel-weights @Radek
6. Work with content team on a mix of activities to fix the channel weight feed @Dmitry (starting from Wednesday)
